src/ui/analyse/forecast/ForecastCtrl.ts

```diff
@@ -0,0 +1,195 @@
+import { keyBy, pickBy, some, values } from 'lodash-es'
```

Comment:
I try to keep my utilisation of lodash as low as possible, and when I use a lodash function, I'd import it like to `import stuff from 'lodash-es/stuff'` to avoid importing everything (tree shaking is not working here I believe).

But here I'd not use any of them (see comments below).